frontend/src/components/Summary/ImplementationRow.css

```diff
@@ -1,4 +1,4 @@
-td{
+td {
   height: 40px;
```

Comment:
I don't think px is the right unit to use here, AFAIK (which isn't much) one should use `rem`s for spacing, but we use bootstrap, so we should presumably be using bootstrap's mechanisms for spacing.

Reply:
Ok! I'll check that out and will let you know!